internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -837,6 +866,46 @@ func (r *AgentServiceConfigReconciler) newAssistedServiceDeployment(instance *ai
 	return deployment, mutateFn
 }
 
+func (r *AgentServiceConfigReconciler) getOpenshiftVersions(instance *aiv1beta1.AgentServiceConfig) string {
+	if instance.Spec.OSImages == nil {
+		return OpenshiftVersions()
+	}
+
+	openshiftVersions := make(models.OpenshiftVersions)
+	for _, image := range instance.Spec.OSImages {
+		v, err := version.NewVersion(image.OpenshiftVersion)
+		if err != nil {
+			r.Log.Error(err, fmt.Sprintf("Problem parsing OpenShift version %v, skipping.", image.OpenshiftVersion))
+			continue
+		}
+
+		// put string in x.y format
+		versionString := fmt.Sprintf("%d.%d", v.Segments()[0], v.Segments()[1])
+		openshiftVersion := models.OpenshiftVersion{
+			DisplayName:  &versionString,
```

Comment:
We can drop the DisplayName property, it's already populated by extracting the version from the release image:
https://github.com/openshift/assisted-service/blob/60c9b8e6f013ec011c2505117ed1263a4adc7e30/internal/versions/versions.go#L243